### PR TITLE
Add SB Integration Test

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
jenkins retest

**Comment by lawrence-lee** [comment]
jenkins retest

File: liferay-gradle/blade.servicebuilder.svc/src/testIntegration/java/com/liferay/blade/samples/servicebuilder/service/test/BladeServiceBuilderIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+/**
+ * @author Lawrence Lee
+ */
+@RunWith(Arquillian.class)
+public class BladeServiceBuilderIntegrationTest {
+
+	@AfterClass
+	public static void cleanBundles() throws Exception {
+		JMXBundleDeployer jmxBundleDeployer = new JMXBundleDeployer(
```
Comment (gamerson) [inline liferay-gradle/blade.servicebuilder.svc/src/testIntegration/java/com/liferay/blade/samples/servicebuilder/service/test/BladeServiceBuilderIntegrationTest.java:48]: Hey lets use this class from this dependency: `biz.aQute.bnd:biz.aQute.remote.api:3.3.0`

File: liferay-gradle/blade.servicebuilder.svc/build.gradle [hunk 2]
```diff
@@ -23,4 +23,28 @@ buildService {
...
+
+startTestableTomcat {
+	checkTimeout = 10 * 60 * 1000
+}
+
+testIntegration {
+	outputs.upToDateWhen { false }
+
+	dependsOn jar, ":initBundle", startTestableTomcat
+	dependsOn ":blade.servicebuilder.api:deploy"
```
Comment (gamerson) [inline liferay-gradle/blade.servicebuilder.svc/build.gradle]: We don't need to both deploy this and then later install it with JMXDeployer.  Lets just use the JMX deployer for now and not the gradle deploy.

File: liferay-gradle/blade.servicebuilder.svc/source-formatter.properties [hunk 3]
```diff
@@ -0,0 +1,3 @@
+source.formatter.excludes=\
+    src/testIntegration/java/com/liferay/blade/samples/servicebuilder/service/test/BladeServiceBuilderIntegrationTest.java,\
```
Comment (gamerson) [inline liferay-gradle/blade.servicebuilder.svc/source-formatter.properties]: Lets go ahead and remove this and just fix the source formatter errors.

File: liferay-workspace/tests/arquillian.extension.lifecycle/src/main/java/org/jboss/arquillian/showcase/extension/lifecycle/LifecycleExecuter.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
+/*
```
Comment (gamerson) [inline liferay-workspace/tests/arquillian.extension.lifecycle/src/main/java/org/jboss/arquillian/showcase/extension/lifecycle/LifecycleExecuter.java:1]: WHat are these classes?  Can we remove them?

**Review by gamerson — CHANGES_REQUESTED** [review]

**Pull request closed without merging**